roles/iptables/templates/iptables-init.bash.j2

```diff
@@ -474,6 +474,27 @@ for EXT_INTERFACE in "${EXTERNAL_INTERFACES[@]:-}"; do
 	done
 	iptables -A OUTPUT -o "${EXT_INTERFACE}" -p tcp --sport 41514 -m state --state ESTABLISHED -j ACCEPT
 {% endif%}
+{% if iptables_allow_ldap_inbound is defined and iptables_allow_ldap_inbound | length > 0 %}
+	#
+	# Allow inbound logs.
+	#
+	# logs requires:
+	#  * TCP port 636: to receive LDAP
```

Comment:
Comment copy pasted from setup for log servers?

Reply:
Fixed. Thanks